### PR TITLE
Integration of Staff Home and HR Home with backend

### PR DESCRIPTION
**Pull request by L1uM4rt1n**
temp, for review later

### CODE REVIEW

**Comment by L1uM4rt1n** [comment]
for your kind reviews :) @ItsPeeko lmk what are the changes required

**Review by ItsPeeko — APPROVED** [review]
alls good boss

**Review by ItsPeeko — APPROVED** [review]
alls good boss

**Review by ItsPeeko — APPROVED** [review]
Looks ok vue part we change when backend linked

**Review by ItsPeeko — APPROVED** [review]
alls good boss

**Review by ItsPeeko — APPROVED** [review]
alls good boss

**Review by ItsPeeko — COMMENTED** [review]
Error occured here in loading staffhome page

**Review by ItsPeeko — APPROVED** [review]

File: Frontend/spm-proj/src/views/Staff/StaffHome.vue [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<template>
+    <div class="app">
+
+        <!-- display of all roles -->
+        <div class="container">
+            <div class="card rounded m-2" style="border: 2px solid #ccc;" v-for="role in filteredRoles" :key="role.Role_ID">
+                <router-link :to="{ name: 'roleListing', params: { id: role.Role_ID } }">
```
Comment (ItsPeeko) [inline Frontend/spm-proj/src/views/Staff/StaffHome.vue]: Error occurred on this line:

No match for
 {"name":"roleListing","params":{"id":"1000002"}}
Error: No match for
 {"name":"roleListing","params":{"id":"1000002"}}
    at createRouterError (webpack-internal:///./node_modules/vue-router/dist/vue-router.mjs:821:19)
    at Object.resolve (webpack-internal:///./node_modules/vue-router/dist/vue-router.mjs:1359:27)
    at Object.resolve (webpack-internal:///./node_modules/vue-router/dist/vue-router.mjs:2899:34)
    at ReactiveEffect.eval [as fn] (webpack-internal:///./node_modules/vue-router/dist/vue-router.mjs:2027:76)
    at ReactiveEffect.run (webpack-internal:///./node_modules/@vue/reactivity/dist/reactivity.esm-bundler.js:224:19)
    at get value [as value] (webpack-internal:///./node_modules/@vue/reactivity/dist/reactivity.esm-bundler.js:1132:33)
    at useLink (webpack-internal:///./node_modules/vue-router/dist/vue-router.mjs:2069:22)
    at setup (webpack-internal:///./node_modules/vue-router/dist/vue-router.mjs:2122:64)
    at callWithErrorHandling (webpack-internal:///./node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js:295:18)
    at setupStatefulComponent (webpack-internal:///./node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js:6522:25)

Comment (L1uM4rt1n) [reply Frontend/spm-proj/src/views/Staff/StaffHome.vue]: removed

**Review by ItsPeeko — APPROVED** [review]
👍

**Pull request merged**